install/uninstall.go

```diff
@@ -50,6 +50,8 @@ func (k *K8sUninstaller) Uninstall(ctx context.Context) error {
 
 	k.Log("🔥 Deleting %s namespace...", k.params.TestNamespace)
 	k.client.DeleteNamespace(ctx, k.params.TestNamespace, metav1.DeleteOptions{})
+	k.Log("🔥 Deleting Ingress Secrets Namespace...")
+	k.client.DeleteNamespace(ctx, defaults.IngressSecretsNamespace, metav1.DeleteOptions{})
```

Comment:
How about combining these in a loop?

```suggestion
	for _, n := range []string{k.params.TestNamespace, defaults.IngressSecretsNamespace} {
		k.Log("🔥 Deleting %s namespace...", n)
		k.client.DeleteNamespace(ctx, n, metav1.DeleteOptions{})
	}
```